algorithm_catalog/max_ndvi_composite_description.md

```diff
@@ -0,0 +1,71 @@
+# Description
+
+This algorithm generates a Sentinel-2 based composite for a selected area and temporal extent. By default, the resolution of the output is 10 meters.
+
+The used compositing method is the "max-NDVI" method, which selects the pixel with the highest NDVI value for each pixel location and within the time window.
+The method falls under the 'rank composite' category, and ensures that selected spectral band values for any individual pixel all come from the same observation.
+
+The method generates good results for timeseries analytics, but spatially neighbouring pixels may be selected from different observations,
+which leads to visual discontinuities in the result.
+
+# Performance characteristics
+
+The method is computationally efficient, as it only requires the B04, B08 and SCL bands to determine the rank score. Loading 
+of other bands can be minimized to read only selected observations.
+
+
+# Examples
+
+The image below shows a typical result over an agricultural area.
```

Comment:
I suggest to add: "Note the typical artifacts due to issues with the S2 SCL cloud masking."